cmd/soroban-rpc/internal/preflight/preflight_test.go

```diff
@@ -362,6 +362,8 @@ func benchmark(b *testing.B, config benchmarkConfig) {
 			disableCache: config.useDB.disableCache,
 		}
 	}
+
+	b.ResetTimer()
 	b.StopTimer()
```

Comment:
To be really accurate the stop should happen before the reset, otherwise the timer will continue to run between the reset and the stop and so won't actually start at in the first iteration of the for loop from zero.

In practice the tiny amount of time it runs for won't be meaningful for two reasons:
- The amount of time should be truly tiny on its own.
- The extra time is added outside the `b.N` loop, and so it'll be spread over hundreds of thousands of iterations essentially making it zero.

Reply:
Uhm, that won't really as is, since there is some preamble work on the loop

Reply:
but, as you said, the difference is tiny